From 3051cfbed63f8f8865b4a3374a474031dc916faa Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 10 Dec 2025 13:47:00 +0000
Subject: [PATCH] Update common submodule

Pull in this change:

  Richard W.M. Jones (1):
      mlcustomize/firstboot.ml: Print %USERNAME% and %USERDOMAIN%

(cherry picked from commit 7ee462f9f9c2235b2d3dacbf3cd1006060108635)
---
 common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule common b54ba2031..d03dd8547:
diff --git a/common/mlcustomize/firstboot.ml b/common/mlcustomize/firstboot.ml
index 360c33d67..c1738e749 100644
--- a/common/mlcustomize/firstboot.ml
+++ b/common/mlcustomize/firstboot.ml
@@ -295,6 +295,7 @@ exit /b
 
 :main
 echo starting firstboot service
+echo effective user: %USERNAME% domain: %USERDOMAIN%
 
 if not exist "%%scripts_done%%" (
   mkdir "%%scripts_done%%"
